jina/executors/encoders/multimodal/__init__.py

```diff
@@ -0,0 +1,37 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
```

Comment:
@bwanglzu btw, no need to add these two lines, they are added by our ci automatically

Reply:
ok, thanks :)